packages/backend/src/open_payments/wallet_address/routes.ts

```diff
@@ -41,6 +41,13 @@ export async function getWalletAddress(
   const walletAddress = await deps.walletAddressService.getOrPollByUrl(
     ctx.walletAddressUrl
   )
+  if (walletAddress && ctx.fetchAdditionalProperties) {
+    walletAddress.additionalProperties =
```

Comment:
we are setting `additionalProperties` here but then it gets lost later when we do `walletAddress.toOpenPaymentsType()`. the `toOpenPaymentsType` method on the `WalletAddress` model should be updated to include `additionalProperties`.

Reply:
Good catch! I also added a test case for this in `routes.test.ts`.

Reply:
Test case is catching it. 

## Run:
https://github.com/interledger/rafiki/actions/runs/9400698822/job/25890897174?pr=2752